src/main/java/gov/ca/emsa/service/controller/PatientDiscoveryController.java

```diff
@@ -128,8 +133,29 @@ public String patientDiscovery(@RequestBody String request) throws InterruptedEx
 					patientPerson.getValue().getAdministrativeGenderCode().setCode("UN");
 				}
 				
-				LocalDate dob = LocalDate.parse(search.getDob(), DateTimeFormatter.ISO_DATE_TIME);
-				patientPerson.getValue().getBirthTime().setValue(dob.format(DateTimeFormatter.BASIC_ISO_DATE));
+				patientPerson.getValue().getBirthTime().setValue(search.getDob());
+				if(!StringUtils.isEmpty(search.getTelephone())){
+					TELExplicit tel = new TELExplicit();
+					tel.setValue("tel:+1-" + search.getTelephone());
```

Comment:
Is this assuming that a person doing a search won't put in the "tel:+1-" line? If they do, would it return something bad? I don't think there's any real reason to make this more complicated than it needs to be, but am curious.